cmd/example/main.go

```diff
@@ -0,0 +1,42 @@
+package main
```

Comment:
@itsjustdel This is basically the starting point of how to integrate self update in a binary. There is also Fyne specific helper in https://github.com/fynelabs/fyneselfupdate .